lib/engine/game/base.rb

```diff
@@ -226,6 +226,8 @@ class Base
 
       MUST_BUY_TRAIN = :route # When must the company buy a train if it doesn't have one (route, never, always)
 
+      DOUBLE_SIDED_TILES = false # if true, requires @tile_groups to be set
```

Comment:
just set tile groups in all and check the emptyness of that instead of this constant

Reply:
Ok